### PR TITLE
x11-*/*: maint-needed EAPI bumps

### PR DESCRIPTION
**Pull request by hanetzer**
Most of these are flat data (icon and sound themes, etc), with a few having actual executables.

All compile tested through `ebuild $ebuild clean install` on my current system, all maint-needed.

### CODE REVIEW

File: x11-themes/wm-icons/wm-icons-0.4.0_pre1-r1.ebuild [hunk 1]
```diff
@@ -27,26 +29,25 @@ src_unpack() {
 	sed -i 's/\t#/#/' "${S}"/Makefile.am
```
[the remark this answers is not in the document]
Comment (hanetzer) [reply x11-themes/wm-icons/wm-icons-0.4.0_pre1-r1.ebuild]: Good idea.

File: x11-themes/wm-icons/wm-icons-0.4.0.ebuild [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
 }
 
 src_install() {
-	make icondir="${D}/usr/share/icons/wm-icons" DESTDIR="${D}" install
+	# strange makefile...
+	emake icondir="${ED%/}/usr/share/icons/wm-icons" DESTDIR="${D}" install
 
 	einfo "Setting default aliases..."
-	"${D}"/usr/bin/wm-icons-config --force --user-dir="${D}/usr/share/icons/wm-icons" --defaults
+	"${ED%/}/usr/bin/wm-icons-config" --force --user-dir="${ED%/}/usr/share/icons/wm-icons" --defaults
```
Comment (Moiman) [inline x11-themes/wm-icons/wm-icons-0.4.0.ebuild]: Missing die

File: x11-themes/wm-icons/wm-icons-0.4.0_pre1-r1.ebuild [hunk 3]
```diff
@@ -1,52 +1,46 @@
...
 }
 
 src_install() {
 	# strange makefile...
-	einstall icondir="${D}/usr/share/icons/wm-icons" DESTDIR="${D}" || die
+	emake icondir="${ED%/}/usr/share/icons/wm-icons" DESTDIR="${D}" install
 
-	dodir /usr/bin
-	mv "${D}"/"${D}"/usr/bin/wm-icons-config "${D}"/usr/bin/wm-icons-config
 	rm -rf "${D}"/var
```
Comment (Moiman) [inline x11-themes/wm-icons/wm-icons-0.4.0_pre1-r1.ebuild]: missing die

File: x11-themes/wm-icons/wm-icons-0.4.0_pre1-r1.ebuild [hunk 4]
```diff
@@ -1,52 +1,46 @@
...
 
 	einfo "Setting default aliases..."
-	"${D}"/usr/bin/wm-icons-config --user-dir="${D}/usr/share/icons/wm-icons" --defaults
+	"${ED%/}/usr/bin/wm-icons-config" --force --user-dir="${ED%/}/usr/share/icons/wm-icons" --defaults
```
Comment (Moiman) [inline x11-themes/wm-icons/wm-icons-0.4.0_pre1-r1.ebuild]: missing die

File: x11-themes/commonbox-styles/commonbox-styles-0.6.ebuild [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
 		http://mkeadle.org/distfiles/${P}.tar.bz2"
-HOMEPAGE="http://mkeadle.org/distfiles/"
 
-SLOT="0"
 LICENSE="GPL-2"
+SLOT="0"
 KEYWORDS="alpha amd64 hppa ia64 ~mips ppc ppc64 sparc x86 ~x86-fbsd"
+IUSE=""
 
 RDEPEND="|| ( x11-wm/fluxbox x11-wm/blackbox x11-wm/openbox )"
```
[the remark this answers is not in the document]
Comment (Moiman) [reply x11-themes/commonbox-styles/commonbox-styles-0.6.ebuild]: Can you remove first and last empty line from src_install?

File: x11-libs/fox-wrapper/fox-wrapper-3.ebuild [hunk 6]
```diff
@@ -1,6 +1,8 @@
-# Copyright 1999-2013 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+EAPI=6
+
 DESCRIPTION="wrapper for fox-config to manage multiple versions"
 HOMEPAGE="https://www.gentoo.org/"
 SRC_URI=""
```
Comment (Moiman) [inline x11-libs/fox-wrapper/fox-wrapper-3.ebuild]: newexe doesn't need die

File: x11-themes/pulse-glass/pulse-glass-20100616.ebuild [hunk 7]
```diff
@@ -16,5 +18,5 @@ S="${WORKDIR}/${MY_PN}"
```
Comment (Moiman) [inline x11-themes/pulse-glass/pulse-glass-20100616.ebuild]: HOMEPAGE="https://store.kde.org/content/show.php/Pulse+Glass?content=124442"
Also fix SRC_URI

File: x11-plugins/gkrellm-plugins/gkrellm-plugins-2.0.ebuild [hunk 8]
```diff
@@ -1,6 +1,8 @@
-# Copyright 1999-2012 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+EAPI=6
+
 DESCRIPTION="emerge this package to install all of the gkrellm plugins"
 HOMEPAGE="http://www.gkrellm.net/"
```
Comment (Moiman) [inline x11-plugins/gkrellm-plugins/gkrellm-plugins-2.0.ebuild]: http://gkrellm.srcbox.net/

**Pull request closed without merging**